install.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+for main in index.{"/usr/bin/git","/usr/bin/unzip","/usr/bin/wget"}
+do
+    command -v "$main" || deps=false
+done
+
+function install-theme()
+{
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Get the latest release of SFP and unzip it
+mkdir SFP
+cd SFP || exit 1
+wget -q https://github.com/PhantomGamers/SFP/releases/download/0.0.14/SFP_UI-SelfContained-linux-x64.zip
+unzip ./*.zip
+chmod +x SFP_UI
+./SFP_UI
+
+# Install metro
+if [ -d ~/.steam ]
+then
+  rm -fr ~/.steam/steam/skins/catppuccin
```

Comment:
We should warn before doing this, and make a backup. We should also install into the directory "catppuccin-[flavor]"-- that way people can install multiple flavors and switch